perl/ANNOTATION.md

```diff
@@ -0,0 +1,781 @@
+# Annotation Fields (Human Assembly hg38 and hg19)
+
+<h2>Annotation Field Description</h2>
+
+##### _Italicized fields_ are custom Bystro fields. All others are sourced as described.
+
+<br/>
+
+#### Summary of the Bystro Annotator
+
+Bystro takes 1+ VCF or SNP (PEMapper/PECaller) files, performs quality control, generates statistics, and outputs an annotated file that is in an easy-to-parse tab-separated format.
```

Comment:
1+ is a little bit of a confusing way to read this. I might just write out 'one or more'